pkg/artifactory/resource_artifactory_local_repository_test.go

```diff
@@ -56,7 +56,6 @@ resource "artifactory_local_repository" "terraform-local-test-repo-full" {
 	calculate_yum_metadata          = false
 	yum_root_depth                  = 0
 	docker_api_version              = "V2"
-	force_nuget_authentication		= true
```

Comment:
You can't enable `forceNugetAuthentication` on a npm repository. I'm not sure why this ever worked or why this was added.